### PR TITLE
Update the Online one-click setup description to be more precise.

### PR DESCRIPTION
**Pull request by nisarhassan12**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ For example, in the string "Import from $1$", we replace $1$ with "Instagram" to
 
 ## Online one-click setup
 
-With the one-click online setup with Gitpod (a free only VS Code-like IDE), you can get a ready-to-code workspace with all the dependencies installed, and websevers via `npm run demo` running, so that you can started quickly and make a pull request from within your browser.
+You can use Gitpod (A free online VS Code like IDE) for contributing online. With a single click it'll launch a workspace, where the repo is already cloned and automatically: dependencies are installed, and `npm run demo` started so that you can start quickly and make a pull request from within your browser.
```
Comment (AnandChowdhary) [inline CONTRIBUTING.md]: Do you think we can add some small grammar changes?

```You can use Gitpod (a free VS Code-like IDE) for contributing online. With a single click, it launches a workspace where the repo is cloned, dependencies are installed, and the `npm run demo` script executed. This helps you get started quickly and make a pull request from within your browser.```

Comment (nisarhassan12) [reply CONTRIBUTING.md]: Thanks! your suggested change looks much better :slightly_smiling_face.

The truth is "dependencies are not installed" at the moment when the workspace is opened that's why I added "automatically". I'm not sure whether removing `automatically` clarifies this?

I'm not that good at English what do you say?

**Comment by nisarhassan12** [comment]
What about this:

image

**Comment by AnandChowdhary** [comment]
That's good too, can you just make sure we have all the small things covered, like "VS Code-like" instead of "VS Code like" that I pointed out? :)

**Comment by nisarhassan12** [comment]
> That's good too, can you just make sure we have all the small things covered, like "VS Code-like" instead of "VS Code like" that I pointed out? :)

Done!

**Comment by AnandChowdhary** [comment]
@all-contributors add @nisarhassan12 for documentation

**Pull request merged**